Source/Core/VideoCommon/UberShaderPixel.cpp

```diff
@@ -525,12 +522,8 @@ ShaderCode GenPixelShader(APIType api_type, const ShaderHostConfig& host_config,
     // intermediate value with multiple reads & modifications, so we pull out the "real" output
     // value above and use a temporary for calculations, then set the output value once at the
     // end of the shader.
-    out.Write("  float4 ocol0;\n");
-  }
-
-  if (use_shader_blend)
-  {
-    out.Write("  float4 ocol1;\n");
+    out.Write("  float4 ocol0;\n"
+              "  float4 ocol1;\n");
```

Comment:
Feels like we should only be defining this if `use_dual_source` is true

Reply:
By this I mean `ocol1` because if `use_dual_source` is true, that variable will be defined already.

Reply:
fbfetch and dual source should be mutually exclusive (since fbfetch should take the place of external blending)

I can add an assertion though

Reply:
I added an assertion, does that look fine?  If so, I'll squash it in.

Reply:
Seems fine to me, squash away!